os/net/lwip/src/include/lwip/api.h

```diff
@@ -84,52 +84,52 @@ extern "C" {
  * this temporarily stores whether to wake up the original application task
  * if data couldn't be sent in the first try.
  */
-#define NETCONN_FLAG_WRITE_DELAYED            0x01
+#define NETCONN_FLAG_WRITE_DELAYED 0x01
 /* Should this netconn avoid blocking? */
-#define NETCONN_FLAG_NON_BLOCKING             0x02
+#define NETCONN_FLAG_NON_BLOCKING 0x02
 /* Was the last connect action a non-blocking one? */
-#define NETCONN_FLAG_IN_NONBLOCKING_CONNECT   0x04
+#define NETCONN_FLAG_IN_NONBLOCKING_CONNECT 0x04
 /*
  * If this is set, a TCP netconn must call netconn_recved() to update
  * the TCP receive window (done automatically if not set).
  */
-#define NETCONN_FLAG_NO_AUTO_RECVED           0x08
+#define NETCONN_FLAG_NO_AUTO_RECVED 0x08
 /*
  * If a nonblocking write has been rejected before, poll_tcp needs to
  * check if the netconn is writable again
  */
-#define NETCONN_FLAG_CHECK_WRITESPACE         0x10
+#define NETCONN_FLAG_CHECK_WRITESPACE 0x10
 #if LWIP_IPV6
 /*
  * If this flag is set then only IPv6 communication is allowed on the
  * netconn. As per RFC#3493 this features defaults to OFF allowing
  * dual-stack usage by default.
  */
-#define NETCONN_FLAG_IPV6_V6ONLY              0x20
-#endif							/* LWIP_IPV6 */
+#define NETCONN_FLAG_IPV6_V6ONLY 0x20
+#endif /* LWIP_IPV6 */
 
 /* Helpers to process several netconn_types by the same code */
-#define NETCONNTYPE_GROUP(t)         ((t) & 0xF0)
-#define NETCONNTYPE_DATAGRAM(t)      ((t) & 0xE0)
+#define NETCONNTYPE_GROUP(t) ((t)&0xF0)
+#define NETCONNTYPE_DATAGRAM(t) ((t)&0xE0)
 #if LWIP_IPV6
-#define NETCONN_TYPE_IPV6            0x08
-#define NETCONNTYPE_ISIPV6(t)        (((t) & NETCONN_TYPE_IPV6) != 0)
-#define NETCONNTYPE_ISUDPLITE(t)     (((t) & 0xF3) == NETCONN_UDPLITE)
-#define NETCONNTYPE_ISUDPNOCHKSUM(t) (((t) & 0xF3) == NETCONN_UDPNOCHKSUM)
-#else							/* LWIP_IPV6 */
-#define NETCONNTYPE_ISIPV6(t)        (0)
-#define NETCONNTYPE_ISUDPLITE(t)     ((t) == NETCONN_UDPLITE)
+#define NETCONN_TYPE_IPV6 0x08
+#define NETCONNTYPE_ISIPV6(t) (((t)&NETCONN_TYPE_IPV6) != 0)
+#define NETCONNTYPE_ISUDPLITE(t) (((t)&0xF3) == NETCONN_UDPLITE)
+#define NETCONNTYPE_ISUDPNOCHKSUM(t) (((t)&0xF3) == NETCONN_UDPNOCHKSUM)
```

Comment:
In my opinion, they are bit operation, not address operator so that previous is right.

Reply:
@sunghan-chang 
I fixed some code.